tutorial/README.md

```diff
@@ -1,85 +0,0 @@
-# Getting Started
-
```

Comment:
Hello, thank you so much for the translations. I think you have mistakenly deleted some of the english files in this PR. Please fix this PR so it does not delete anything and then I can merge in your translations.